a-braham/Questioner
# #163346865 Added date field to meetup form and made page responsive

#### What does this PR do?
 * The pull request modifies meetup.html page

#### Description of Task to be completed?
  **Have the following features complete**
  * The form should be simple
  * Page should be responsive
  * Should not have any inline css

#### How should this be manually tested?
  * Clone the repository
  * Open the admin/meetup.html file with chrome
  * Test for responsiveness, check the tags for errors
 
#### What are the relevant pivotal tracker stories?
  #163346865

#### Screenshots
![screenshot from 2019-01-20 10-15-31](https://user-images.githubusercontent.com/45788436/51436351-5ea72900-1c9c-11e9-8f7c-fdfd5cee719d.png)
